libs/newsletter/package.json

```diff
@@ -13,6 +13,9 @@
     "type": "git",
     "url": "https://github.com/graycoreio/daffodil"
   },
+  "publishConfig": {
+    "directory": "../../dist/newsletter"
```

Comment:
Why is this needed?

Reply:
It's not this was added trying to fix an import issue but I didn't remove it.